### ISSUE TITLE
Newsletter_send.create error 400

### ISSUE THREAD

**Issue by hirokipf** [mailjet/mailjet-gem#67]
Hi everyone,

When I try to send newsletter, I receive the following error:

```
Mailjet::ApiError: error 400 while sending #<RestClient::Resource:0x007fa56de90e48 @url="https://api.mailjet.com/v3/REST/newsletter/7755/send", @block=nil, @options={:public_operations=>[:post], :read_only=>nil, :user=>"HASHOFUSER", :password=>"HASHOFPASSWORD", :content_type=>"application/json"}> to https://api.mailjet.com/v3/REST/newsletter/7755/send with {"Id"=>7755}

"{ \"ErrorInfo\" : \"\", \"ErrorMessage\" : \"Invalid json input: object \\\"\\\"->\\\"TNewsLetterSend\\\" property \\\"Id\\\" is not a class property, but tkInt64\", \"StatusCode\" : 400 }"

Please see http://api.mailjet.com/0.1/HelpStatus for more informations on error numbers.

from /path-to-gem/lib/mailjet/connection.rb:83:in `handle_exception'
```

Here is my code :

```
result = Mailjet::Newsletter_send.create(id: newsletter_id)
```

Thanks!

**Comment by GuillaumeBadi** [comment]
This one should be solved on 1.3.8

Sorry about that, we had a refactor issue, please tell me if it works on your side :+1: 

Guillaume

**Comment by hirokipf** [comment]
It worked on 1.3.8.
Thank you for your swift reply!
